corehq/apps/app_manager/views/utils.py

```diff
@@ -128,10 +131,14 @@ def get_blank_form_xml(form_name):
     })
 
 
-def overwrite_app(app, master_build, report_map=None):
+def overwrite_app(app, master_build, report_map=None, maintain_ids=False):
     excluded_fields = set(Application._meta_fields).union(
         ['date_created', 'build_profiles', 'copy_history', 'copy_of', 'name', 'comment', 'doc_type']
     )
+    if maintain_ids:
+        id_map = _get_form_id_map(app)
+    else:
+        id_map = {}
```

Comment:
nit: the only place I see this getting used is on line 159 where it's passed to `_update_form_ids`. That line is also under the same if check `if maintain_ids` so it seems to me like `id_map` can be defined there:
```
if maintain_ids:
    id_map = _get_from_id_map(app)
    wrapped_app = _update_form_ids(wrapped_app, master_build, id_map)
```